### PR TITLE
Shift request tab to org

### LINKED ISSUE #1876: ADMIN REDESIGN: Move Requests tab to be Organization specific
**Is your feature request related to a problem? Please describe.**
Currently there is a combined `Requests` tab for the admins, it should be different for each Organization.

**Describe the solution you'd like**
Shift the Requests tab to each Organization screen.

**Describe alternatives you've considered**
NA

Currently:
- It's on common screen for `/requests`

Design:
- Add "Requests" below "People"

image

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by pranshugupta54**
**What kind of change does this PR introduce?**

- Moving "Requests" tab to Organization specific for Admins
- routesReducer uses INITIAL_STATE instead of state passed in function, fixed this

**Issue Number:**

Fixes #1876

**Did you add tests for your changes?**

Yes

**Snapshots/Videos:**

image

**If relevant, did you update the documentation?**

NA

**Summary**
- **Requests** tab is now Organization specific 
- Allowed Superadmins to view requests too
- Fixed types for files

**Does this PR introduce a breaking change?**

NA

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

### CODE REVIEW

**Comment by pranshugupta54** [comment]
This PR also fixed routesReducer file for fixing INITIAL_STATE being used instead of `state` passed

**Comment by pranshugupta54** [comment]
Introspection fail is not related to this PR

**Comment by pranshugupta54** [comment]
Codecov shows `-4` lines because reduced total lines of code from `5040` to `5036` but coverage is 98.35% only

File: src/screens/Requests/Requests.tsx [hunk 1]
```diff
@@ -143,7 +141,7 @@ const Requests = (): JSX.Element => {
   };
   /* istanbul ignore next */
```
Comment (aashimawadhwa) [inline src/screens/Requests/Requests.tsx:142]: remove this comment and add test for this code too.

**Review by aashimawadhwa — CHANGES_REQUESTED** [review]

File: src/screens/Requests/Requests.tsx (hunk 1 above)
Comment (aashimawadhwa) [reply src/screens/Requests/Requests.tsx:142]: @pranshugupta54

Comment (pranshugupta54) [reply src/screens/Requests/Requests.tsx:142]: Ohk

Comment (pranshugupta54) [reply src/screens/Requests/Requests.tsx:142]: @aashimawadhwa, I've removed the ignore part for 3 codes. But I'm not able to write tests for `loadMoreRequests` function as it's a major bug in InfiniteScroll package which gives issue for loading new data on scroll on different sizes. (It's not being maintained for 3 years+)

**Comment by palisadoes** [comment]
1. Move the requests card between `funds` and `plugins`.
2. It's going to be a relatively infrequently used option

**Comment by pranshugupta54** [comment]
Fixed 
image

**Pull request merged**